pom.xml

```diff
@@ -556,6 +573,22 @@
                 <artifactId>exec-maven-plugin</artifactId>
                 <version>${exec-maven-plugin.version}</version>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>buildnumber-maven-plugin</artifactId>
+                <version>${buildnumber-maven-plugin.version}</version>
+                <configuration>
```

Comment:
Since the `configuration` and the `executions` are already defined in the plugin in `pluginManagement` they can be removed